packages/suite/src/actions/wallet/send/sendFormEthereumActions.ts

```diff
@@ -149,6 +149,13 @@ export const composeTransaction =
             }
         } else {
             // TODO: catch error from blockbook/geth (invalid contract, not enough balance...)
+            if (estimatedFee.payload.error) return;
+            dispatch(
+                notificationsActions.addToast({
+                    type: 'metadata-unexpected-error',
```

Comment:
Please create a new type something like `estimated-fee-error` and provide a general message saying that it failed.